### PR TITLE
Fixed switch custom color with disabled.

### PR DESCRIPTION
**Pull request by gogu**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

`switch` 组件 `disabled` 状况下自定义颜色不正常，如此例中：http://jsfiddle.net/j95axgwh/

- `mounted` 时 `disabled` 的组件 enable 后自定义颜色不生效
- 一旦触发自定义颜色生效则 `disabled` 样式的颜色不生效

### CODE REVIEW

**Comment by furybean** [comment]
gogu 又开始进这个坑了 😉

File: packages/theme-default/src/switch.css [hunk 1]
```diff
@@ -80,15 +80,15 @@
 
     @when disabled {
       .el-switch__core {
-        border-color: var(--switch-disabled-color);
-        background: var(--switch-disabled-color);
+        border-color: var(--switch-disabled-color) !important;
```
Comment (QingWei-Li) [inline packages/theme-default/src/switch.css]: 这样似乎不容易去覆盖样式了？

Comment (gogu) [inline packages/theme-default/src/switch.css]: 我理解的话，`disabled` 状态就应该是 important 级别的样式了
……恩，如果加上 `disabled-on-color`，`disabled-off-color` 这样的配置，然后和 `on-color`，`off-color` 一样统一在元素行内进行覆盖，也许合理点？

Comment (Leopoldthecoder) [inline packages/theme-default/src/switch.css]: 我觉得王老师这样改可以。

不过两个例子还是不要改了吧。

**Pull request merged**